index.js

```diff
@@ -35,6 +36,12 @@ module.exports = {
     });
   },
 
+  ignoreAutoRotate(shouldIgnore) {
+    if (Platform.OS === 'android') {
+      Orientation.ignoreAutoRotate(shouldIgnore);
```

Comment:
Just checking, this means we had the react-native-bridge API for ignoring auto rotate already, but there wasn't a JS API for this? If so, it makes sense to me to add this now. I'm curious about why we didn't have the JS API before.

Reply:
Yeah that's correct. I am also curious...not sure why.

Reply:
@mrkcsc might have some context why since he re-wrote the android component of this library